src/CyBy/Draw/Extensions/Word.idr

```diff
@@ -0,0 +1,39 @@
+||| This module takes care of all events used for the word add-in (extension)
+module CyBy.Draw.Extensions.Word
+
+import Web.MVC
+
+import CyBy.Draw.Event
+import CyBy.Draw.MoleculeCanvas
+import CyBy.Draw.Internal.Settings
+import CyBy.Draw.Internal.Graph
+import CyBy.Draw.Extensions.Util
+
+%default total
+
+-- uses the Word-API for exporting an svg string to a word document image
+%foreign "browser:lambda:(s,w) => {Word.run(async (context) => {const b64 = btoa(s); const docSelection = context.document.getSelection(); docSelection.insertInlinePictureFromBase64(b64, Word.InsertLocation.end); context.sync();});}"
+prim__exportImageToWord : String -> PrimIO ()
+
+-- uses the Word-API for extracting an svg string from a word document image
+%foreign "browser:lambda:(f,w) => {Word.run(async context => {const selection = context.document.getSelection(); selection.load('inlinePictures'); await context.sync(); const items = selection.inlinePictures.items; if (items.length > 0) { const b64 = await items[0].getBase64ImageSrc(); await context.sync(); const valB64 = b64.value; await context.sync(); f(atob(valB64))(w); } else f('')(w); });}"
```

Comment:
Could you please document this some more? Espcecially, could you please explain why all the calls to `await context.sync()` are required and what they do? Please note also that it should be possible to use multi-line strings with the JS backends.